3723/0/about_dice_project.rb

```diff
@@ -0,0 +1,72 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class DiceSet
+  attr_accessor :round, :values
+
+  def initialize
+    @values = []
+    @round = round
+  end
+
+  def roll(num)
+    @values = []
```

Comment:
why do you need it here? if you initialized it during creating object, you don't need to initialize it in each method.

Reply:
Fixed